src/static/webcomponents/video/video.html

```diff
@@ -0,0 +1,152 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
```

Comment:
Note to create a new issue to swap out the video web component with the MjpegVideo web component that was written as a validation for using web components.